specs/expectToBe.spec.js

```diff
@@ -0,0 +1,39 @@
+/*global describe, expect, require, beforeEach, browser, xit, it, element, by, beforeAll*/
+
+(function () {
+    "use strict";
+
+    var ExpectHelperPage = require('../helper/expectHelper.js'),
+        expectHelperPage = new ExpectHelperPage(),
+        
```

Comment:
I don;t see the issue here.. Am i missing something.. A little more details pls..